### ISSUE TITLE
Goal of this project?

### ISSUE THREAD

**Issue by rvirding** [lfex/mnemosyne#2]
I was just wondering where we want to go with this project? LFE already supports QLCs. There is even the `lfe_trans` module for translating LFE <-> Erlang which is used by this[*]. I have nothing against having another interface if we wish.

[*] Using this would simplify the compiler but where's the fun in that?

**Comment by oubiwann** [comment]
`*chuckles*` (about the footnote)

The goals of this project are to provide:
- an API (modules + function names) that are familiar/expected to the average web developer (I'm afraid I'm going to have to look at MongoDB!)
- a unified set of module names (wrappers for mnesia and qlc) that provide a consistent developer experience for those using the API

**Comment by arpunk** [comment]
I ended up having some sort of wrapper around mnesia aswell (github.com/arpunk/memento), mostly for moving the params order to provide a consistent usage and minor syntax sugar to be a little more like [mito](https://github.com/fukamachi/mito) and [sxql](https://github.com/fukamachi/sxql).

In the end LFE offered the QLC wrappers and macros that I needed, so I stopped working on my wrapper from the time being.

Being that said, I think that in the projects I've used mnesia I was happy I didn't had to use any wrappers to have a more straight forward translation from the Erlang world.
